packages/web/src/index.html

```diff
@@ -4,33 +4,37 @@
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0, minimum-scale=1.0, maximum-scale=5.0" />
     <title>Stencil Component Starter</title>
+
+    <!-- Utility framework -->
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/cds-snc/gcds-utilities/dist/utilities.css" />
+
+
```

Comment:
Importing first so that this demo page can overwrite utilities. For example, font awesome would load from utilities, instead of from this index. Usually isn't it better to place utilities first?